charts/sysdig/templates/daemonset.yaml

```diff
@@ -303,5 +303,7 @@ spec:
                 fieldPath: metadata.name
               path: name
   updateStrategy:
-{{ toYaml .Values.daemonset.updateStrategy | indent 4 }}
+    type: RollingUpdate
+    rollingUpdate: 
+      maxUnavailable: {{include "sysdig.parallelStarts" .}}
```

Comment:
I'm assuming the customers won't specify the maxSurge value

Reply:
I think having it there is a good idea, since it's only these two values in the [spec](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.23/#rollingupdatedaemonset-v1-apps) at the moment.

something like `maxSurge: {{ .Values.daemonset.updateStrategy.rollingUpdate.maxSurge }}`

Reply:
noted

Reply:
shall I do a null check?